services/aws-roles.ts

```diff
@@ -1,9 +1,15 @@
 import { instance } from './axios';
 
-export async function GetUsersPaginatedList(licencePlate: string, role: string, currentPage: string, pageSize: string) {
+export async function GetUsersPaginatedList(
```

Comment:
I believe we have naming condition for JS function name as `camelCase`?

Reply:
sure,  changed